lib/view/pages/connections/connections_page.dart

```diff
@@ -177,7 +177,8 @@ class _FriendSearchState extends State<FriendSearch> {
         title: const Text('Friends'),
```

Comment:
We should maybe also make this text 'connections'.

Reply:
this a good idea

Reply:
I will quickly change it. I got pretty confused at a stage with all the renaming 😅